### DISCUSSION TITLE
Útskriftarferðin - 2024/5/0

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#256] [vefsida-ummæli]
# Útskriftarferðin - 2024/5/0

Heimasíða Biskupsreglunnar

https://www.biskupsreglan.com/verkefni/%C3%9Atskriftafer%C3%B0

**Comment by MatthiasGudni** [comment]
Illa lyktandi tær 🔥🔥

**Reply by MatthiasGudni** [reply]
Þorskar 😳😳
